src/field_control_avatar.c

```diff
@@ -182,6 +206,13 @@ int ProcessPlayerFieldInput(struct FieldInput *input)
 
     GetInFrontOfPlayerPosition(&position);
     metatileBehavior = MapGridGetMetatileBehaviorAt(position.x, position.y);
+
+    if (input->heldDirection && input->dpadDirection == playerDirection)
+    {
+        if (TrySetUpWalkIntoSignpostScript(&position, metatileBehavior, playerDirection) == TRUE)
```

Comment:
Is there a reason this isn't just nested in the previous condition?

Reply:
I _assume_ ghoul was just following the rest of the convention of the rest of this function, but an easy fix.

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5044/commits/b66678248364f134e32f07b7608d816c57f0e4cb